### DISCUSSION TITLE
sMite - Spells not applying properly

### DISCUSSION THREAD

**Discussion by Sagimoster** [thatskymod/Sky-CotL-Scripts#55] [Q&A]
This Sky update comes out that a problem sMite not working properly 

https://github.com/thatskymod/Sky-CotL-Scripts/assets/149201564/6bf2cf72-1abc-452e-a7f9-b7ea3d6a18ee

**Comment by thatskymod** [comment]
### Closing as not fixable / impossible to resolve.

The ability to fake or substitute spells was a feature in the past but is no longer supported due to restrictions imposed by thatgamecompany (i.e., the server now validates ownership of spells, preventing any possibility of spell forgery).

Thank you for your report, and have a great day!
